docker/cron/generate_sitemap/generate_and_upload.sh

```diff
@@ -8,7 +8,7 @@ BASE_URL_PATH="${BASE_URL:=https://test.osv.dev}"
 
 echo "Begin sitemap generation for $BASE_URL_PATH"
 
-./generate_sitemap.py --base_url $BASE_URL_PATH
+generate_sitemap.py --base_url $BASE_URL_PATH
```

Comment:
Doesn't this get copied to `/usr/local/bin/generate_sitemap/generate_sitemap.py`? I don't think that's in the PATH

Reply:
Huh, good point, I'm not sure why I was so sure it would be on the $PATH. I'll just put the full path there.